umami/gp.py

```diff
@@ -56,14 +55,11 @@ def dist2(ls, x1, x2=None):
 def grad_dist2(ls, x1, x2=None):
     if x2 is None:
         x2 = x1
-        
+
     # Rescale.
     x1 = x1 / ls
     x2 = x2 / ls
-    
-    N = x1.shape[0]
```

Comment:
Ah, dang. Good catch, I'll back these lines out tomorrow and add # noqa or something.